application/models/services/ThemeQuestionAttributeProvider.php

```diff
@@ -41,7 +41,7 @@ protected function getAttributesFromQuestionTheme($questionThemeName, $questionT
 
         $questionTheme = \QuestionTheme::model()->findByAttributes([], 'name = :name AND extends = :extends', ['name' => $questionThemeName, 'extends' => $questionType]);
         if ($questionTheme !== null) {
-            $xmlFilePath = $questionTheme['xml_path'] . '/config.xml';
+            $xmlFilePath = $questionTheme->getRelativeXmlPath() . '/config.xml';
```

Comment:
There must be other locations where xml_path is read??

Reply:
Maybe not needed to replace all other spots.
We could, but would replace it by some other method, like getXmlPath().
I have left a comment on Zoho.

Reply:
Hm, are you saying absolute path is OK in some places but not others? :thinking:

Reply:
Yes. Absolute and "LS root based paths" can be ok when used internally. Actually, it is needed as to read the config files.
Still, when exposing the paths, as in hidden files, that's where we want to "split" the absolute path in:
- relative path
- type of plugin

Usign those 2 components, later when processing the hidden fields, the path can be located.

Reply:
Well, first task is to make CI green.

Reply:
Let's make LS green again!

Reply:
:cow: :notes: 

Happy cow :D